.github/actions/integration-tests/action.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Integration tests'
+description: 'Run integration tests'
+inputs:
+  github-token:
+    description: 'The GITHUB_TOKEN secret'
+    required: true
```

Comment:
Is it a requirement and/or are there benefits of requiring this as an input vs. using the system var directly? asking as it seems like both "call sites" are just passing the same secret

Reply:
An action doesn't automatically inherit secrets/vars from the workflow that calls it. You have to explicitly pass them in as params. This makes a lot of sense from a security pov if you are running third-party actions in your workflow and you want to send them some variables but not everything.